### PR TITLE
Show all trainee and Trainer of course

### PR DESCRIPTION
**Pull request by DatDaiGia**

### CODE REVIEW

File: app/views/supervisor/courses/show.html.erb [hunk 1]
```diff
@@ -53,6 +53,40 @@
...
+            </div>
+            <div class="tt-trainer">
+              <h4><%= t ".trainer_in_course" %></h4>
+              <div class="tt-trainer-list">
+                <% @user_courses.each do |user_courses| %>
+                  <div class="media">
+                    <div class="media-body">
+                      <% if user_courses.user.supervisor? %>
+                        <h6 class="media-heading">
+                        <%= link_to user_courses.user.name,
```
Comment (dieunb) [inline app/views/supervisor/courses/show.html.erb]: indent

File: app/views/supervisor/courses/show.html.erb [hunk 2]
```diff
@@ -53,6 +53,40 @@
...
+          <div class="col-xs-12 col-sm-4">
+            <div class="tt-trainee">
+              <h3><%= t ".trainee_in_course" %></h3>
+              <div class="tt-trainee-list">
+                <% @user_courses.each do |user_courses| %>
+                  <div class="media">
+                    <div class="media-body">
+                      <% if user_courses.user.trainee? %>
+                        <h6 class="media-heading">
+                        <%= link_to user_courses.user.name,
```
Comment (dieunb) [inline app/views/supervisor/courses/show.html.erb]: indent

**Comment by TaoLuyen** [comment]
ok

**Comment by hangxuka** [comment]
ok

**Comment by hellovietnam93** [comment]
OK

**Pull request merged**